utils/build.ps1

```diff
@@ -879,7 +936,7 @@ function Build-Compilers($Arch, [switch]$Test = $false) {
         LLVM_TABLEGEN = "$BinaryCache\0\bin\llvm-tblgen.exe";
         LLVM_USE_HOST_TOOLS = "NO";
         SWIFT_BUILD_SWIFT_SYNTAX = "YES";
-        SWIFT_CLANG_LOCATION = "$BinaryCache\toolchains\$PinnedToolchain\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin";
+        SWIFT_CLANG_LOCATION = Get-PinnedToolchainTool "";
```

Comment:
Why the concatenation with `""`?

Reply:
Because `Get-PinnedToolchainTool` took an argument as the last component of the path. I changed it to return the path prefix like the other two.